.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libsodium-sys/libsodium"]
+	path = libsodium-sys/libsodium
+	url = https://github.com/jedisct1/libsodium.git
```

Comment:
Shouldn't it be pinned to 1.0.16 tag?

Reply:
There is no such mechanism to pin by tag, only by commit. See:

```
 Submodule libsodium added at 675149 
```

Which equals to 1.0.16

Reply:
ok, it seems you are right.